src/Enum.php

```diff
@@ -121,10 +122,42 @@ public static function isValid($value) : bool
      *
      * @return array
      */
-    protected static function getDeclaredConstants() : array
+    protected static function getDeclaredConstants(): array
     {
         $reflection = new ReflectionClass(get_called_class());
 
         return (array) $reflection->getConstants();
     }
+
+    /**
+     * @return array
+     */
+    public static function toSelect(): array
+    {
+        return static::collect()
+            ->mapWithKeys(function ($key) {
+                $outcome = self::ofType($key);
+
+                return [$outcome->value() => $outcome->line()];
+            })
+            ->toArray();
+    }
+
+    /**
+     * @return string
+     */
+    public function line(): string
+    {
+        $vendor = $this->vendor() ?: null;
```

Comment:
Because the method returns null, we can remove this and just use `$this->vendor()`